README.md

```diff
@@ -134,111 +134,65 @@ command with `--cap-add IPC_LOCK` or `--ulimit memlock=-1:-1`. See
 [here](https://docs.tigerbeetle.com/deployment/with-docker#error-systemresources-on-macos) for
 more information.
 
-### Use Node as a CLI
+### Using the CLI
```

Comment:
Maybe "Client CLI"?  To disambiguate from `format`/`start`/etc.

Reply:
Yep great point, I've been saying CLI Client in all the other docs, will change to that.